src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewRoundedCell.swift

```diff
@@ -0,0 +1,97 @@
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import UIKit
+
+class DynamicTableViewRoundedCell: UITableViewCell {
+
+	// MARK: - View elements.
+
+	lazy var title = UILabel(frame: .zero)
+	lazy var body = UILabel(frame: .zero)
+	lazy var insetView = UIView(frame: .zero)
+
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+	}
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+	}
+
+	override func awakeFromNib() {
+		super.awakeFromNib()
+		self.autoresizingMask = .flexibleHeight
+	}
+
+	private func setup() {
+
+		// MARK: - General cell setup.
+		selectionStyle = .none
+		backgroundColor = .preferredColor(for: .backgroundPrimary)
+
+		// MARK: - Add inset view
+		insetView.backgroundColor = .preferredColor(for: .backgroundSecondary)
+		insetView.layer.cornerRadius = 16.0
+
+		// MARK: - Title adjustment.
+
+		title.font = .preferredFont(forTextStyle: .headline)
+		title.lineBreakMode = .byWordWrapping
+		title.numberOfLines = 0
+
+		// MARK: - Body adjustment.
+		body.font = .preferredFont(forTextStyle: .body)
+		body.lineBreakMode = .byWordWrapping
+		body.numberOfLines = 0
```

Comment:
Should be handled by `ENALabel` as @ChristianKienle mentioned above.
Also the label color `.textPrimary1` is missing.